tests/robot/_resources/keywords/directory_keywords.robot

```diff
@@ -51,6 +51,14 @@ create DIRECTORY (JSON)
 
                         POST /ehr/ehr_id/directory    JSON
 
+                        Log  TO CLARIFY: @AXEL - version_uid format???  level=WARN
```

Comment:
@axelsiebert  are you aware of this?

Reply:
There is currently no handing of full version_uid implemented. Thus these tests might fail.

An issue for future implementations has been created: https://github.com/ehrbase/project_management/issues/48